### PR TITLE
Generate file with all images

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

**Comment by lulf** [comment]
@enmasse-ci run tests profile=systemtests-upgrade

File: systemtests/scripts/test_func.sh [hunk 1]
```diff
@@ -390,20 +390,11 @@ function is_upgraded() {
     TAG=${TAG:-"latest"}
 
     IMAGE_TAG=${IMAGE##*:}
-    TEMPLATES=$(find ${CURDIR}/../../templates/build/enmasse-${TAG} -name '*.yaml' -exec cat {} \; | grep "image")
```
Comment (kornys) [inline systemtests/scripts/test_func.sh]: @lulf not sure how it can work, because you don't read content of file

Comment (lulf) [reply systemtests/scripts/test_func.sh]: Doh, fixed :)

**Comment by lulf** [comment]
@enmasse-ci run tests profile=systemtests-upgrade

**Comment by lulf** [comment]
@enmasse-ci run tests profile=systemtests-upgrade

**Pull request merged**